tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -501,6 +502,100 @@ class ExecutionTest {
                        L"Table:ShaderOpArithTable.xml#PackUnpackOpTable")
   END_TEST_METHOD()
 
+  BEGIN_TEST_METHOD(LongVector_BinaryOpTest_bool)
+  TEST_METHOD_PROPERTY(L"DataSource",
+                       L"Table:ShaderOpArithTable.xml#LongVector_BinaryOpTable")
+  END_TEST_METHOD()
+
+  //BEGIN_TEST_METHOD(LongVector_BinaryOpTest_float16)
+  //TEST_METHOD_PROPERTY(L"DataSource",
+  //                     L"Table:ShaderOpArithTable.xml#LongVector_BinaryOpTable")
+  //END_TEST_METHOD()
```

Comment:
What's up with float16?

Reply:
Needed to add logic because C++ doesn't have a native type for it. Well, C++23 does.